src/data/BackendInterface.cpp

```diff
@@ -326,6 +326,8 @@ BackendInterface::fetchLedgerPage(
                 msg << " - " << ripple::strHex(keys[j]);
             }
             LOG(gLog.error()) << msg.str();
+            LOG(gLog.error()) << "Disabling the cache to avoid corrupting the DB further. Please investigate.";
```

Comment:
Will not this significantly slow down clio?
If yes, I think we need to notify about it more loud than just in the log. I would suggest to add Prometheus metric here and also add a warning to server_info response. WDYT?

Reply:
Add a flag in server_info +1
And highlight the consequence of corruption is incorrect data for ledger_data and book_offers

Reply:
Yes . It will slow down the Clio. Another solution will be only disable cache for ETL side.  
https://github.com/XRPLF/clio/blob/develop/src/etl/impl/LedgerFetcher.hpp#L99
The cache will be still workable for RPC side. But data is incorrect.

Reply:
Well, you are definitely right that we should highlight it more.. i'll cook something up.